mobile/Features/Journal/Components/JournalCard.jsx

```diff
@@ -0,0 +1,28 @@
+import {
+    ScrollView, Text, View, Button, TextInput, Keyboard, TouchableWithoutFeedback, Modal, TouchableOpacity, Pressable,
+  } from 'react-native';
+  import PropTypes from 'prop-types';
+  import React, { useState } from 'react';
+  import axios from 'axios';
+  import styles from './JournalHistoryStyle';
+  
+  //props: username, prompt, text, timestamp
+  export default function JournalCard(props) {
+    const handlePress = (text) => {
+        props.onPress(text);
+    } //calls the onPress function from props, which is a function in JournalHistoryPage to navigate to the past journal entry with the previously written text
```

Comment:
Another thing, now that es-lint is working for you, try to avoid these super long lines that eslint throws warnings for